docroot/themes/custom/uids_base/uids_base.theme

```diff
@@ -426,6 +426,7 @@ function uids_base_preprocess_layout(&$variables) {
       $publish_options = $node->get('field_publish_options')->getValue();
       if (array_search('title_hidden', array_column($publish_options, 'value')) !== FALSE) {
         // For uiowa_bootstrap/classy default.
+        // @todo Can this be removed?
         $title_hidden = TRUE;
```

Comment:
@joewhitsitt or @bspeare Can you weigh in?

Reply:
i don't think this one can be removed. this is setting a publish option that sitenow.profile preprocess block then references and sets element-invisible for uids_base page_title_block

Reply:
I should have included the line above in the snippet here, it is this:
```php
// For uiowa_bootstrap/classy default.
```

Reply:
i think that is an old comment because work was later done to get this working for page_title_block.

https://github.com/uiowa/uiowa/commit/9d1b0c5f1d9855c2230bff6293715c11e055edbd